roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -53,6 +53,10 @@
     with_items:
       - "OVESETUP_DWH_DB/password=str:{{ he_dwh_db_password }}"
     when: he_dwh_db_password is defined
+  - name: Setup Repositories on Local VM
+    import_role:
+      name: ovirt.ovirt.repositories
+    when: he_apply_repositories_role|bool
```

Comment:
You don't need to specify the `|bool` filter. 
The filter is used to convert string `"True"` -> `True`
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#forcing-the-data-type